deployments/production/spark-2-1-0-worker-daemonset.json

```diff
@@ -35,12 +33,19 @@
                                 "name": "SPARK_WORKER_MEMORY",
                                 "valueFrom": {
                                     "resourceFieldRef": {
-                                        "divisor": "0",
                                         "resource": "requests.memory"
                                     }
                                 }
                             },
                             {
+                                "name": "SPARK_WORKER_CORES",
+                                "valueFrom": {
+                                    "resourceFieldRef": {
+                                        "resource": "requests.cpu"
+                                    }
+                                }
+                            },
+                            {
                                 "name": "SPARK_MASTER_HOST",
                                 "value": "spark-cluster-master-2-1-0"
```

Comment:
per my other comment - I believe we will need to update these values `spark-cluster-master-2-1-0` -> `spark-master-2-1-0` if the dns resolution is changing

Reply:
Service name (dns hostname) has not changed (see my other comment).